tests/queries/0_stateless/03203_system_query_metric_log.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+# shellcheck source=../shell_config.sh
+. "$CURDIR"/../shell_config.sh
+
+readonly query_prefix=$CLICKHOUSE_DATABASE
+
+$CLICKHOUSE_CLIENT --query-id="${query_prefix}_1000" -q "SELECT sleep(2.5) FORMAT Null" &
+$CLICKHOUSE_CLIENT --query-id="${query_prefix}_1234" -q "SELECT sleep(2.5) SETTINGS query_metric_log_interval=1234 FORMAT Null" &
+$CLICKHOUSE_CLIENT --query-id="${query_prefix}_123" -q "SELECT sleep(2.5) SETTINGS query_metric_log_interval=123 FORMAT Null" &
+$CLICKHOUSE_CLIENT --query-id="${query_prefix}_0" -q "SELECT sleep(2.5) SETTINGS query_metric_log_interval=0 FORMAT Null" &
+$CLICKHOUSE_CLIENT --query-id="${query_prefix}_fast" -q "SELECT sleep(0.1) FORMAT Null" &
+
+wait
+
+$CLICKHOUSE_CLIENT -q "SYSTEM FLUSH LOGS"
+
+function check_log()
+{
+    interval=$1
+    # We calculate the diff of each row with its previous row to check whether the intervals at which
+    # data is collected is right. The first row is always skipped because the diff is 0. The same for the
```

Comment:
`The first row is always skipped because the diff is 0`. Here you mean it's because the preceding row is itself so it will be 0, right? Just confirming

Reply:
Done at [`4fc4cee` (#66532)](https://github.com/ClickHouse/ClickHouse/pull/66532/commits/4fc4cee8627eae133c18d22fcd297d1445922eb8)